### PR TITLE
Document that GitHub username is added to internal.logins

### PR DESCRIPTION
**Pull request by stevenGravy**
Was not documented that GitHub username is added to `internal.logins` trait for user.

### CODE REVIEW

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 1]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the 
+session. The GitHub username is automatically added to that user's `internal.logins` variable
+usable within roles. The default `access` role has this variable that would automatically include the
```
Comment (alexfornuto) [inline docs/pages/access-controls/sso/github-sso.mdx:126]: would implies "used to".

```suggestion
usable within roles. The default `access` role has this variable that will automatically include the
```

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 2]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the
```
Comment (zmb3) [inline docs/pages/access-controls/sso/github-sso.mdx:124]: I don't love this sentence.

- "Teleport will create a user within Teleport" is a little confusing
- "GitHub type user" is not a term I'd expect the reader to understand

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 3]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the 
+session. The GitHub username is automatically added to that user's `internal.logins` variable
```
Comment (zmb3) [inline docs/pages/access-controls/sso/github-sso.mdx:125]: Technically that's a _trait_, not a variable, right?

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 4]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
...
+usable within roles. The default `access` role has this variable that will automatically include the
```
Comment (zmb3) [inline docs/pages/access-controls/sso/github-sso.mdx:126]:
```suggestion
usable within roles. The preset `access` role has this variable that will automatically include the
```

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 5]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
...
+GitHub user as an authorized SSH login. Here is an example role configuration snippet using the variable.
```
Comment (zmb3) [inline docs/pages/access-controls/sso/github-sso.mdx:127]:
```suggestion
GitHub user as an authorized SSH login. Here is an example role configuration snippet using the trait. 
```

File: docs/pages/access-controls/sso/github-sso.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/access-controls/sso/github-sso.mdx]: Removed and simplified language.

File: docs/pages/access-controls/sso/github-sso.mdx (hunk 3 above)
Comment (stevenGravy) [reply docs/pages/access-controls/sso/github-sso.mdx]: The `internal.logins` is the trait to the user.  Within here we refer to `internal.logins` as a variable for its usage in roles.  I've updated to describe as a trait variable that I believe is more clear.

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 6]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will add the GitHub username
+to that user's `internal.logins` trait for their session. The preset `access`
+role has this trait variable configured to include the GitHub user as an
+authorized SSH login. Here is an example role configuration snippet using the trait variable.
```
Comment (alexfornuto) [inline docs/pages/access-controls/sso/github-sso.mdx:127]: Adjusted line breaks :

```suggestion
After authentication Teleport will add the GitHub username to that user's
`internal.logins` trait for their session. The preset `access` role has this
trait variable configured to include the GitHub user as an authorized SSH login.
Here is an example role configuration snippet using the trait variable. 
```

**Review by alexfornuto — APPROVED** [review]

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 7]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will add the GitHub username to that user's
```
Comment (ptgott) [inline docs/pages/access-controls/sso/github-sso.mdx:124]:
```suggestion
After a user authenticates, Teleport will add the user's GitHub username to their
```

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 8]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
...
+`internal.logins` trait for their session. The preset `access` role has this
```
Comment (ptgott) [inline docs/pages/access-controls/sso/github-sso.mdx:125]:
```suggestion
`internal.logins` trait for their Teleport session. The preset `access` role has this
```

File: docs/pages/access-controls/sso/github-sso.mdx [hunk 9]
```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
...
+trait variable configured to include the GitHub user as an authorized SSH login.
+Here is an example role configuration snippet using the trait variable.
```
Comment (ptgott) [inline docs/pages/access-controls/sso/github-sso.mdx:127]:
```suggestion
Here is an example role configuration snippet using the trait variable:
```

**Review by ptgott — APPROVED** [review]
Approved with suggestions. Disabled auto-merge so you can apply them

**Pull request merged**